Vagrantprov

```diff
@@ -0,0 +1,30 @@
+ #! /bin/bash
+
+# Ensure apt is up to date
```

Comment:
There is quite some overlap between this and .travis.yml's install section. Maybe we could move this to the makefile and just call it from both places?